docs/ui/workspaces.md

```diff
@@ -0,0 +1,104 @@
+## Create a workspace
+
+On the **Workspaces** page, select the **+** icon to create a new workspace. You'll be prompted to configure:
+
+- The workspace owner &mdash; the user account or organization managing the workspace.
+- A handle, or name, for the workspace. This name must be unique within your account or organization.
+- An optional description for the workspace.
+
+![Creating a new workspace in the Prefect Cloud UI.](/img/ui/create-workspace.png)
+
+Select **Create** to actually create the new workspace. The number of available workspaces varies by [Prefect Cloud plan](https://www.prefect.io/pricing/). See [Pricing](https://www.prefect.io/pricing/) if you need additional workspaces or users. 
+
+## Workspace settings
+
+Within a workspace, select **Workspace Settings** to view or edit workspace details.  
+
+![Managing a workspace in the Prefect Cloud UI.](/img/ui/workspace-settings.png)
+
+This page enables you to copy the configuration string to [set up a local development or execution environment](/ui/cloud-getting-started/#configure-execution-environment) to work this workspace.
+
+The options menu enables you to edit workspace details or delete the workspace.
+
+!!! warning "Deleting a workspace"
+    Deleting a workspace deletes all deployments, flow run history, work queues, and notifications configured in workspace.
+
+## Workspace collaborators
+
+Personal account users may invite _workspace collaborators_, users who can join, view, and run flows in your workspace.
+
+In your workspace, select **Workspace Collaborators**. If you've previously invited collaborators, you'll see them listed.
+
+![Managing collaborators in a workspace in the Prefect Cloud UI.](/img/ui/workspace-collaborators.png)
+
+To invite a user to become a workspace collaborator, select the **+** icon. You'll be prompted for the email address of the person you'd like to invite. Add the email address, then select **Send** to initiate the invitation. 
+
+If the user does not already have a Prefect Cloud account, they will be able to create one when accepting the workspace collaborator invitation.
+
+To delete a workspace collaborator, select **Remove** from the menu on the left side of the user's information on this page.
+
+## Workspace sharing <span class="badge orgs"></span>
+
+Within a Prefect Cloud [organization](/ui/organizations/) organization, Admins and workspace Owners may invite users and [service accounts](/ui/service-accounts/) to work in an organization workspace. In addition to giving the user access to the workspace, the Admin or Owner assigns a [workspace role](/ui/roles/) to the user. The role specifies the scope of permissions for the user within the workspace.
+
+!!! tip "Workspace access in Prefect Cloud organizations"
+    Within an organization, users who are not Admins do not have any default access to organization workspaces. An organization Admin or workspace Owner _must_ invite users to organizations and apply workspace roles to users.
```

Comment:
This is actually configurable on workspace sharing, the default access for users. Its service accounts that dont get the privileges conferred by default workspace access.